post.md

```diff
@@ -1,273 +0,0 @@
-ncurses library). First in `Mainboard` menu we set platform:
-
-```
-Mainboard vendor (Intel)  --->
-Mainboard model (Minnow Max)  --->
-```
-
-Except of this we have to configure microcode and enable FSP in `Chipset` menu:
-
-```
-Include CPU microcode in CBFS (Include external microcode header files)  --->
-(M0130673321.h  M0130679901.h  M023067221D.h) List of space separated microcode header
-
-[*] Use Intel Firmware Support Package
-(../intel/fsp/baytrail/BAYTRAIL_FSP.fd) Intel FSP binary path and filename (NEW)
-```
-
-If we want to use VgaBIOS, we can configure it in `Devices`:
-
-```
- [*] Add a VGA BIOS image
-(../intel/cpu/baytrail/vbios/Vga.dat) VGA BIOS path and filename (NEW)
-```
-
-On x86 by default, coreboot chooses `SeaBIOS` as a payload so that it provides
-legacy BIOS interface. You may want to use one of other options: GRUB (well
-known, robust Open Source boot loader), Tianocore (UEFI), etc.
-
-This is minimal configuration for MinnowBoard. However you probably want
-to decrease log level (defualt 8:SPEW is much too verbose) in `Console`
-menu:
-
-```
-Default console log level (5: NOTICE)  --->
-```
-
-Build & Flash
--------------
-
-When configuration is complete, we can save it and run:
-
-```
-make
-```
-
-If everything goes well, output should end like this:
-
-```
-    CBFSPRINT  coreboot.rom
-
-Name                           Offset     Type           Size   Comp
-cbfs master header             0x0        cbfs header        32 none
-fallback/romstage              0x80       stage           30204 none
-cpu_microcode_blob.bin         0x7700     microcode           0 none
-fallback/ramstage              0x7780     stage           58614 none
-config                         0x15cc0    raw               118 none
-revision                       0x15d80    raw               582 none
-cmos_layout.bin                0x16000    cmos_layout      1208 none
-fallback/dsdt.aml              0x16500    raw             12528 none
-fallback/payload               0x19640    payload         68158 none
-payload_config                 0x2a0c0    raw              1593 none
-payload_revision               0x2a740    raw               239 none
-(empty)                        0x2a880    null          2969432 none
-bootblock                      0x2ff800   bootblock        1720 none
-    HOSTCC     cbfstool/ifwitool.o
-    HOSTCC     cbfstool/ifwitool (link)
-
-Built intel/minnowmax (Minnow Max)
-```
-
-So we can flash it. If you are using docker image, remember that it doesn't
-contain `flashrom` so you have to do that outside container:
-
-```
-sudo flashrom -l mb.layout -i bios -p dediprog -w build/coreboot.rom
-```
-
-Conclusion
-----------
-
-This procedure is pretty straightforward, but in practice turns out to cause
-much trouble at first time. It also covers only most basic options. We are
-open to help if you have problem with that. Also if you don't want to do that
-we provide such a service.
```

Comment:
@mscibisz are you sure you want to remove this file?

Reply:
Actually, it is the file that was overwritten previously. It seems I didn't have the updated version of post.md I will name it properly and put it in _posts folder. Thanks for checking